Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/FeedAzureADConnectHealth_test.py

```diff
@@ -0,0 +1,22 @@
+from AzureADConnectHealthFeed import Client
+
+
+URL = 'https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-health-agent-install#outbound-connectivity-to-the-azure-service-endpoints' # noqa
+
+
+def test_build_iterator(requests_mock):
```

Comment:
can you add a docstring? e.g:

    """Unit test
    Given
    - get_indicators_command func
    - command args
    - command raw response
    When
    - mock the Client's get_stix_objects.
    Then
    - convert the result to human readable table
    - create the context
    valid